doc/book/admin/_includes/1.6-to-2.x-condition.rst

```diff
@@ -1,3 +1,8 @@
+Versions later that 1.6 have incompatible :ref:`.snap <internals-snapshot>` and
+:ref:`.xlog <internals-wal>` file formats: 1.6 files are
+supported during upgrade, but you won’t be able to return to 1.6 after running
+under 1.7 for a while. A few configuration parameters are also renamed.
```

Comment:
"under 1.10 / 2.x"

That's up to you, but I think there's no reason to mention unsupported Tarantool versions